### ISSUE TITLE
`Uri.uri` macro fails with symbol named org

### ISSUE THREAD

**Issue by rossabaker** [http4s/http4s#704]
Well, this is sad.

``` scala
scala> import org.http4s._

scala> Uri.uri("http://www.example.com/")
res0: org.http4s.Uri = http://www.example.com

scala> val bar = "bar"
bar: String = bar

scala> Uri.uri("http://www.example.com/") / bar
res1: org.http4s.Uri = http://www.example.com/bar

scala> val org = "foo"
org: String = foo

scala> Uri.uri("http://www.example.com/") / org
<console>:16: error: value http4s is not a member of String
       Uri.uri("http://www.example.com/") / org
              ^
```

**Comment by rossabaker** [comment]
This was fixed by #741.

**Comment by samspills** [comment]
This one has resurfaced

Running scala-cli with `scala-cli repl --scala 2.13.15 --toolkit org.typelevel:default`

```scala
scala> import org.http4s.implicits._
import org.http4s.implicits._

scala> uri"org"
val res0: org.http4s.Uri = org

scala> val org = "blerf"
val org: String = blerf

scala> uri"org" / org
       ^
       error: value http4s is not a member of String
```
